File: 240413/알파벳에 따른 평가/evaluation-by-alphabet.js
const fs = require('fs');
const alpha = fs.readFileSync(0).toString();

let result;

if(alpha === 'S'){
    result = 'Superior';
}else if(alpha === 'A'){
    result = 'Excellent';
}else if(alpha === 'B'){
    result = 'Good';
}else if(alpha === 'C'){
    result = 'Usually';
}else if(alpha === 'D'){
    result = 'Effort';
}else{
    result = 'Failure';
}

console.log(result);

File: 240413/알파벳에 따른 평가/evaluation-by-alphabet.java
import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        String alpha = sc.next();
        String result;


        if(alpha.equals("S")){
            result = "Superior";
        }else if(alpha.equals("A")){
            result = "Excellent";
        }else if(alpha.equals("B")){
            result = "Good";
        }else if(alpha.equals("C")){
            result = "Usually";
        }else if(alpha.equals("D")){
            result = "Effort";
        }else{
            result = "Failure";
        }

        System.out.println(result);
        // System.out.println(alpha);
    }
}